src/tribol/mesh/MeshData.cpp

```diff
@@ -694,75 +588,75 @@ void MeshData::computeNodalNormals( int const dim )
       allocIntArray( &numFaceNrmlsToNodes, size, 0 );
    } // end if-check on null mesh
 
-   // loop over cells
-   for (int i=0; i<this->m_numCells; ++i)
+   // loop over elements
+   for (int i=0; i<this->numberOfElements(); ++i)
    {
-      // loop over cell nodes
```

Comment:
Minor: going back to the comment of cells vs elements vs faces vs facets let's just establish a convention and stick with it. Various ways to look at it but my mind always thinks of the finite "elements" of the calculation as the underlying volume elements. Cells is more generic, and can be the faces of a surface mesh. These are cells or faces rather than the host code's finite "elements". This also sticks with lexicon familiar to other codes. This is nuanced with no right answer so let's discuss.